#Traversability message
std_msgs/Header header
float64[] areas
float64[] slopes
float64[] scores
mesh_msgs/MeshGeometryStamped mesh
geometry_msgs/Point[] normals
geometry_msgs/Point[] baricenters
geometry_msgs/Point sight_dir
traversability_mesh_package/Proximals[] neighbors
traversability_mesh_package/Features[] face_features
traversability_mesh_package/AccMetrics[] metrics